src/nvme/types.h

```diff
@@ -491,6 +491,30 @@ enum nvme_cmbsts {
 
 #define NVME_CMBSTS_CBAI(cmbsts)	NVME_GET(cmbsts, CMBSTS_CBAI)
 
+enum nvme_unit {
+	NVME_UNIT_B,
```

Comment:
Since these are types with fixed values, it's better to explicit assign them the values, e.g. `NVME_UNIT_B = 1` as we do in the other types.

Reply:
Fixed as mentioned. Thank you.